test/FunctionalTests/ConfigTests/ConfigCustomProvidersWebTests.cs

```diff
@@ -0,0 +1,75 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Server.Testing;
+using Microsoft.AspNetCore.Testing.xunit;
+using Xunit;
+
+namespace EntropyTests.ConfigTests
+{
+    public class ConfigCustomProvidersWebTests
+    {
+        private const string SiteName = "Config.CustomProviders.Web";
+
+        [Theory]
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, "http://localhost:6200")]
+        public async Task RunSite_AllPlatforms(ServerType server, RuntimeFlavor runtimeFlavor, RuntimeArchitecture architecture, string applicationBaseUrl)
+        {
+            await RunSite(server, runtimeFlavor, architecture, applicationBaseUrl);
+        }
+
+        [ConditionalTheory]
+        [OSSkipCondition(OperatingSystems.Linux)]
+        [OSSkipCondition(OperatingSystems.MacOSX)]
+        //[InlineData(ServerType.WebListener, RuntimeFlavor.Clr, RuntimeArchitecture.x86, "http://localhost:6201")]
```

Comment:
Can we mark these as skipped with a reason? Like [here](https://github.com/aspnet/Entropy/blob/6cdf157595047a2a6a50c7785fc58bb57b9cd4d4/test/FunctionalTests/Diagnostics/DiagnosticsStatusCodesMvcTests.cs#L23). That way there's at least a notification that we have skipped tests rather than just them not existing.